classes/actions/ShoppingfeedOrderImportActions.php

```diff
@@ -1401,4 +1404,30 @@ protected function addJustEnoughStock($id_product, $id_product_attribute, $refer
             $this->getIdShop()
         );
     }
+
+    protected function addDniToAddress(Address $address, OrderResource $apiOrder)
+    {
+        $orderArray = $apiOrder->toArray();
+
+        if (empty($orderArray['additionalFields']['buyer_identification_number'])) {
+            return $address;
+        }
+
+        if (empty(Address::$definition['fields']['dni']['validate'])) {
+            return $address;
+        }
+
+        $dni = $orderArray['additionalFields']['buyer_identification_number'];
+        $validateMethod = Address::$definition['fields']['dni']['validate'];
+
+        if (false == is_callable(['Validate', $validateMethod])) {
+            return $address;
```

Comment:
You should add a log to say the DNI xoxo is not valid regarding PrestaShop validation rule. Data cannot be added to this address ?